packages/react/src/primitives/CheckboxField/CheckboxField.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Checkbox } from '../Checkbox';
+import { FieldErrorMessage, FieldDescription } from '../Field';
+import { Flex } from '../Flex';
+import { CheckboxFieldProps } from '../types/checkboxField';
+import { ComponentClassNames } from '../shared';
+
+export const CheckboxField: React.FC<CheckboxFieldProps> = ({
+  alignContent,
+  alignItems,
+  checked,
+  children,
+  className,
+  defaultChecked,
+  descriptiveText,
+  direction = 'column',
+  errorMessage,
+  gap,
+  hasError = false,
+  id,
+  isEmphasized,
+  isDisabled,
+  isReadOnly,
+  isRequired,
+  justifyContent,
+  label,
+  labelHidden = false,
```

Comment:
Is there a use case for even hiding a checkbox label? There's no other way to hint at what the checkbox is for. I think we should ignore this prop entirely, and remove it once we fix the `[key: any]: string` issue.

Reply:
Oh I see, it's used for the description. I guess it will go away when you remove that.